Extensions/BitmapText/bitmaptextruntimeobject-pixi-renderer.js

```diff
@@ -145,33 +106,45 @@ gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.updateFont = function () {
     .getPIXITexture(bitmapTextureResourceName);
 
   // Get the bitmap font file and use the texture for generate the bitmapFont (PIXI.BitmapFont) and return the fontName ready to use.
-  this._object._runtimeScene
+  const bitmapFont = this._object._runtimeScene
     .getGame()
     .getBitmapFontManager()
-    .loadBitmapFont(bitmapFontResourceName, texture)
-    .then((fontName) => {
-      this._pixiObject.fontName = fontName;
-    })
-    .catch((error) => {
-      console.error('Error while loading a bitmapFont resource:', error);
-    });
+    .getBitmapFontFromData(bitmapFontResourceName, texture);
+  this._pixiObject.fontName = bitmapFont.font;
+  this._pixiObject.fontSize = bitmapFont.size;
 
   this.updatePosition();
 };
 
-gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.updateFontSize = function () {
-  this._bitmapFontStyle.fontSize = this._object._fontSize;
-  this._pixiObject.fontSize = this._object._fontSize;
-  this._pixiObject.fontName = this._ensureFontAvailableAndGetFontName(
-    this._pixiObject.fontName
+gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.updateTint = function () {
+  this._pixiObject.tint = gdjs.rgbToHexNumber(
+    this._object._tint[0],
+    this._object._tint[1],
+    this._object._tint[2]
   );
+  this._pixiObject.dirty = true;
+};
+
+gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.getTint = function () {
+  return this._pixiObject.tint;
+};
+
+gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.updateScale = function () {
+  this._pixiObject.scale.set(this._object._scale);
+};
+
+gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.getScale = function () {
+  return this._pixiObject.scale.x && this._pixiObject.scale.y; // both should be equal, return the biggest
```

Comment:
You sure about this? It won't return the biggest, it will return this._pixiObject.scale.y if this._pixiObject.scale.x is not 0.